packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -150,8 +152,36 @@ export class Create {
         if (options && options.responseTimeout != null) {
             headers.push({[ZosmfHeaders.X_IBM_RESPONSE_TIMEOUT]: options.responseTimeout.toString()});
         }
+
         const tempOptions = JSON.parse(JSON.stringify({ like: likeDataSetName, ...(options || {}) }));
         Create.dataSetValidateOptions(tempOptions);
+
+        /*
+        *  This is a fix for issue https://github.com/zowe/vscode-extension-for-zowe/issues/2610.
+        *
+        *  If no block size is passed, then retrieve the attributes of the "Like" dataset and
+        *  set the block size, as the zosmf Rest API does not set the block size properly in
+        *  some instances.
+        *
+        */
+        if (tempOptions.blksize === null || tempOptions.blksize === undefined) {
+            let likeDataSetObj: IZosmfListResponse;
+            const likeDataSetList  = await List.dataSet(session, likeDataSetName, {
+                attributes: true, maxLength: 1,
+                start: likeDataSetName,
+                recall: "wait"
```

Comment:
Thanks for considering waiting for a recall operation. (source: [_X-IBM-Migrated-Recall_](https://www.ibm.com/docs/en/zos/3.1.0?topic=interface-zos-data-set-member-utilities#IZUHPINFO_API_PutDataSetMemberUtilities__title__5))

Do we know what would happen if the migrated data set is in a volume which requires TSO prompts to be recalled?

This may be a larger endeavor, so I'm not asking for this recall to be handled here.  😋